ui/imports/shared/status/StatusSearchListPopup.qml

```diff
@@ -0,0 +1,166 @@
+import QtQuick 2.14
```

Comment:
Can we use this new component for the other place that `StatusInputListPopup` was used? It can be done in another PR, but it would be nice to not keep both components since they basically do the same thing

Reply:
Good point, I will try to deliver it in the next PR, altogether with `StatusInputListPopup` removal.